tests/errors/e_typing_errors.pyx

```diff
@@ -53,4 +57,10 @@ _ERRORS = """
 20:30: typing.Optional[...] cannot be applied to type MyStruct
 
 28:20: Modifier 'typing.ClassVar' is not allowed here.
+
+30:23: 'object' takes exactly one template argument.
+30:43: 'object' takes exactly one template argument.
+30:65: 'object' takes exactly one template argument.
+30:89: 'object' takes exactly one template argument.
+38:39: 'object' takes exactly one template argument.
```

Comment:
This is a rather unclear error message. We should try to keep the old one, or make the error detection explicit and reject it with a dedicated error message.